mcux/README

```diff
@@ -92,3 +92,5 @@ Patch List:
      support is enabled.
 
    * Add USB device controller drivers, the drivers are based on MCUXpresso SDK release tag REL_2.5.0_REL9_RFP_RC3_7_1.
+
+   * Change 32 to 64 to avoid time delay calculation error caused by realDelay and bestDelay overflow.
```

Comment:
Need to mention the fsl_lpspi driver

Reply:
Thanks for your patient guidance, I have performed these steps, please review again